src/Api/Utilities/ApiHelpers.cs

```diff
@@ -69,4 +70,26 @@ public async static Task<T> ReadJsonFileFromBody<T>(HttpContext httpContext, IFo
 
         return new OkObjectResult(response);
     }
+
+    public static Tuple<DateTime, DateTime> GetDateRange(DateTime? start, DateTime? end)
```

Comment:
Non-blocking suggestion: I know you just moved this logic around into a helper method, but I think it would be super helpful to add some comments about what's going on here.

Reply:
Thank you!